swift/Todo/Model/ObservableQuery.swift

```diff
@@ -32,23 +32,16 @@ class ObservableQuery : ObservableObject {
     
     // Wrapper around Result that conforms to identifiable so we can use it in a SwiftUI List
     struct IResult : Identifiable {
-        public let id: UUID
+        public let id: String
         public let wrappedResult: CouchbaseLiteSwift.Result
```

Comment:
Is Result used by the Swift framework already? This is suck. We probably should change it to QueryResult when we could break the API.